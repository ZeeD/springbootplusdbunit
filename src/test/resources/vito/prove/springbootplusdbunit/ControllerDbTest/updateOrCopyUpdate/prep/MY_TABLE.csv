id,name,something_date
1,"a",1234-05-06 07:08:09
123,"_",1970-01-01 00:00:00
