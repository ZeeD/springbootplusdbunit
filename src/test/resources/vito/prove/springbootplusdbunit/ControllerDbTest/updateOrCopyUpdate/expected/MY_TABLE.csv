id,name,when
1,"a",1234-05-06 07:08:09
123,"newName",2020-11-05 05:58:13 +0000
